lib/mixlib/install/generator/powershell/scripts/install_project.ps1

```diff
@@ -102,8 +104,19 @@ function Install-Project {
 }
 set-alias install -value Install-Project
 
-Function Install-ChefMsi($msi) {
-  $p = Start-Process -FilePath "msiexec.exe" -ArgumentList "/qn /i $msi" -Passthru -Wait
+Function Install-ChefMsi($msi, $addlocal) {
+  if ($addlocal -eq "service") {
+    $p = Start-Process -FilePath "msiexec.exe" -ArgumentList "/qn /i $msi ADDLOCAL=`"ChefServiceFeature`"" -Passthru -Wait
+  }
+  ElseIf ($addlocal -eq "task") {
+    $p = Start-Process -FilePath "msiexec.exe" -ArgumentList "/qn /i $msi ADDLOCAL=`"ChefSchTaskFeature`"" -Passthru -Wait
+  }
+  ElseIf ($addlocal -eq "none") {
+    $p = Start-Process -FilePath "msiexec.exe" -ArgumentList "/qn /i $msi" -Passthru -Wait
+  }
+  Else {
+    throw "Invalid value $addlocal passed for -daemon option. Valid values are task, service and none."
```

Comment:
Use `validateset` in the params for daemon options similar to architectures. No need for this else condition.  I would also recommend using a different name that `none`. The script uses 'auto' elsewhere.